stdlib/3/concurrent/futures/_base.pyi

```diff
@@ -54,3 +58,42 @@ def as_completed(fs: Iterable[Future[_T]], timeout: Optional[float] = ...) -> It
 
 def wait(fs: Iterable[Future[_T]], timeout: Optional[float] = ..., return_when: str = ...) -> Tuple[Set[Future[_T]],
                                                                                                     Set[Future[_T]]]: ...
+
+class _Waiter:
+    event: threading.Event = ...
+    finished_futures: Iterable[Future] = ...
```

Comment:
Should use a concrete `List`. Also, `= ...` is redundant and should be left out. (Also applies to code below.)